packages/build/tests/helpers/config.js

```diff
@@ -0,0 +1,56 @@
+'use strict'
+
+const fastSafeStringify = require('fast-safe-stringify')
+const { getBinPath } = require('get-bin-path')
+
+// Tests require the full monorepo to be present at the moment
+// TODO: split tests utility into its own package
+const { runFixtureCommon } = require('./common')
+
+const netlifyConfigPromise = import('@netlify/config')
+
+const NETLIFY_CONFIG_ROOT_DIR = `${__dirname}/../../../config`
+const BINARY_PATH = getBinPath({ cwd: NETLIFY_CONFIG_ROOT_DIR })
+
+// Run fixture using `@netlify/config`.
+// This is inside `@netlify/build`'s repository since it needs it as well.
+const runFixtureConfig = async function (t, fixtureName, { flags = {}, env, ...opts } = {}) {
```

Comment:
Some of the test codes had to be moved from `@netlify/config` to `@netlify/build` due to being used by both.